lib/ci-helper.ts

```diff
@@ -159,6 +159,8 @@ export class CIHelper {
                        { workDir: this.workDir })).split("\n"),
         );
         let result = false;
+        const bases = new Set<string>();
+        const heads = new Set<string>();
```

Comment:
Does `git commit-tree` reject duplicate `-p commit_sha` parameters?  If not, is there a need to create a set that is converted to an array?

Reply:
It errors out when it sees those, indeed.

Reply:
I added a code comment, see [the interdiff](https://github.com/gitgitgadget/gitgitgadget/compare/c07a3158f20fa90f5418ebce6eeb1d871f61e1f1..baf6d52011de4cd83f2b7176d0971ebc4d1cf3bf).